.github/workflows/update-browserlist.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Browserlist DB
+on:
+  schedule:
+    - cron:  '0 0 * * 1'
+  workflow_dispatch:
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
+        npm: [8.5.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node }}
+    - run: npm install -g npm@${{ matrix.npm }}
+    - run: make requirements
+
+    - name: Update dependencies
+      run: npx browserslist@latest --update-db
+
+    - name: setup testeng-ci
+      run: |
+        git clone https://github.com/edx/testeng-ci.git
```

Comment:
It looks like this PR creator is something that would be widely useful to the Open edX community.  (I mean, it's already going into all MFEs.  :)  Would it be possible for us to get in touch with the maintainer of the `testeng-ci` repository, and then to consider copying the relevant bits into the openedx org?

Reply:
I am not sure about this, @jmbowman can comment on this